client/verta/verta/endpoint/build/_build_scan.py

```diff
@@ -87,6 +87,18 @@ def _get(cls, conn: _utils.Connection, build_id: int):
 
         return cls(response.json())
 
+    @classmethod
+    def _start(cls, conn: _utils.Connection, build_id: int, external: bool) -> "BuildScan":
```

Comment:
Hmmmmm I think we ought to call this `_create` to match our usual get/create terminology.

Reply:
Done